app/components/emoji/emoji.js

```diff
@@ -114,8 +115,14 @@ export default class Emoji extends React.PureComponent {
             textStyle,
             token
         } = this.props;
-        const {fontSize} = StyleSheet.flatten(textStyle);
-        const size = scaleEmojiBasedOnDevice(fontSize);
+
+        let size = this.props.size;
+        let fontSize = size;
+        if (!size && textStyle) {
```

Comment:
Can you take out the `!size`? I think it's a bit cleaner if we have the text style take precedence here over the hardcoded size

Reply:
I was actually thinking the other way, if the size is set respect that

Reply:
I am curious about this. When is an emoji ever bigger than the fontSize. I understand it isn't always set, but shouldn't they be a uniform size? 
My PR's assumption was that the fontSize would always be set, which was wrong, but I feel like we should set the emoji to always be whatever the default text size.

Reply:
emoji picker for instance when you are going to add a reaction

Reply:
I assume the emoji's size should always be uniform though. You shouldn't have a 200x200 emoji showing up
Anyway, I am sure it is fine. 👍